ponder.config.ts

```diff
@@ -13,11 +13,14 @@ const maxBlockRange = process.env.MAX_BLOCK_RANGE
   ? parseInt(process.env.MAX_BLOCK_RANGE)
   : 2_000;
 
+const rpc =
+  process.env.AVALANCHE_RPC || "https://api.avax.network/ext/bc/C/rpc";
```

Comment:
how can we make this multi-chain compatible? I read in ponder docs they name rpcs as RPC_1, RPC_2, so I'd suggest we already introduce a map by chainId